config/hub/manifests/odr/kustomization.yaml

```diff
@@ -10,6 +10,7 @@ configMapGenerator:
     disableNameSuffixHash: true
     labels:
       config.openshift.io/inject-trusted-cabundle: "true"
+      app: ramen-hub
```

Comment:
This change should also be done in:
- https://github.com/red-hat-storage/ramen/blob/0e1953920ed9c003aa1ded8cd108a443f70a2229/config/dr-cluster/manifests/odr/kustomization.yaml#L7-L12
- https://github.com/red-hat-storage/ramen/blob/0e1953920ed9c003aa1ded8cd108a443f70a2229/config/dr-cluster/manifests/idr/kustomization.yaml#L7-L12
- https://github.com/red-hat-storage/ramen/blob/0e1953920ed9c003aa1ded8cd108a443f70a2229/config/hub/manifests/idr/kustomization.yaml#L7-L12

Reply:
added the changes